README.md

```diff
@@ -1 +1,86 @@
 # guitar-shop
+
+## Description
+
```

Comment:
- Guitar  Shop is a web application that gives the users peace of mind about shopping for guitars online.